tests/Integration/Core/Test_TimeUtilities.py

```diff
@@ -0,0 +1,41 @@
+"""Test for the TimeUtilities module
+"""
+
+import pytest
+import datetime
+
+from DIRAC.Core.Utilities import TimeUtilities
+
+
+def test_toEpoch():
+    # Try with a datetime type object
+    dt = datetime.datetime(1997, 6, 12, 2, 30, 0, 0)
+    res = TimeUtilities.toEpoch(dt)
+    assert res == 866082600
+
+    # Let us try now with a date type object
+    dt = datetime.date(1997, 6, 12)
+    res = TimeUtilities.toEpoch(dt)
+    # it should return the timestamp of the date with the time set to midnight
+    assert res == 866073600
+
+    # Now with another type
+    dt = datetime.time()
+    res = TimeUtilities.toEpoch(dt)
+    assert not res["OK"]
```

Comment:
why this not?

Reply:
Calling this function with `datetime.time()` makes no sense so it's okay for it to crash.

Reply:
Do we want the `toEpoch()` to take in also `datetime.time` objects?

Reply:
What would that even mean?

`toEpoch` returns the number of seconds since 1970-01-01 so a time without a date doesn't make sense for this function.